DashAI/front/src/components/custom/DeleteItemModal.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { GridActionsCellItem } from "@mui/x-data-grid";
+import DeleteIcon from "@mui/icons-material/Delete";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from "@mui/material";
+
+function DeleteItemModal({ deleteFromTable }) {
+  const [open, setOpen] = React.useState(false);
+  const handleDelete = () => {
+    deleteFromTable();
+    setOpen(false);
+  };
+  return (
+    <React.Fragment>
+      <GridActionsCellItem
+        key="delete-button"
+        icon={<DeleteIcon />}
+        label="Delete"
+        onClick={() => setOpen(true)}
+        sx={{ color: "#ff8383" }}
```

Comment:
I think it would be better to use the colors that are included in the ui material, so if we change one of these it will not be necessary to look for them in the code.

https://mui.com/system/palette/#color

```suggestion
        sx={{ color: "error.main" }}
```
And:

`import { red } from '@mui/material/colors';`

Reply:
- I updated `DeleteItemModal.jsx` to use "error.main" like in the suggested change.
- Similarly  i updated `EditDatasetModal.jsx` to use "warning.main".
- I added values to `style/theme.js` to define custom colors for "error.main" and "warning.main"